students/csimmons/lesson03/mailroom.py

```diff
@@ -77,29 +79,25 @@ def generate_thankyou(f_response, f_gift):
 def send_thankyou(donorlist):
     donors = list(map(lambda x:x[0], donorlist))
     response = input(thanks_prompt)
-    f_response = response.title()
     if response.lower() == 'list':
         print_donors(donors)
-        main()
-    elif response.lower() == 'exit':
-        main()
+    elif response.lower() == 'exit': 
+        menu()
```

Comment:
these should be `return` statements so that it can exit the function and to return to its original loop.